src/ophys_etl/modules/segmentation/qc/seed.py

```diff
@@ -0,0 +1,75 @@
+import h5py
+import numpy as np
+from pathlib import Path
+from typing import Union, Optional
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+
+def add_seeds_to_axes(
+        figure: Figure,
```

Comment:
I wonder if you can just get rid of `figure` as a parameter to this function altogether.

It seems like the only reason it exists is because `figure.tight_layout()` is called, but that should probably not be called within this function anyway.

Reply:
the reason I included it here, is that I wanted the legend to not be on top of the plot, and implemented that by doing:
```
axes.legend(bbox_to_anchor=(1.05, 1))
```
but, that then exceeds the figure bounds without `tight_layout()`

Reply:
Can't `tight_layout()` just be called on the figure outside of `add_seeds_to_axes()`?

Reply:
yes, it could, but, I want this utility to produce a particular nice-looking plot without needing to specify calling something else to make sure the legend is not covering up part of the plot.
This is not a very low-level routine, it is making a specific plot.